### ISSUE TITLE
Tab order

### ISSUE THREAD

**Issue by michael-shields** [flathub/com.jetbrains.IntelliJ-IDEA-Community#93]
with the last update of IntelliJ, I have noticed that I am unable to change the editor file tab order.  I use this functionality to keep my related code beside each other to allow jumping back and forth, this doesn't work in the current version of IntelliJ 2022.1 (Ultimate) build 221.5080.210 April 11, 2022

**Comment by x80486** [comment]
@michael-shields, I'm not entirely sure what you meant by "change the editor file tab order". Could you elaborate on that?

**Comment by michael-shields** [comment]
Yes, drag to another position, this is a normal functionality in ALL applications with tabs.  Tab reorder you should be able to CLICK, HOLD DOWN and move the tab to another position.  I have confirmed on 2 boxes the same issue with IntelliJ.

Regards,
Michael Shields

On Apr 20, 2022, at 1:53 PM, x80486 ***@***.***> wrote:

﻿

@michael-shields<https://github.com/michael-shields>, I'm not entirely sure what you meant by "change the editor file tab order". Could you elaborate on that?

**Comment by x80486** [comment]
I do have IntelliJ IDEA installed in Arch Linux, and I'm not facing that issue.

It doesn't look like Flatpak-related either.

Would you be able to unzip the latest version from JetBrains and give it a whirl? In the meantime, I will install it as well in a couple of VMs that I have for testing.

**Comment by x80486** [comment]
@michael-shields, feel free to reopen this one if you are still facing the same issue(s).
